src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.util.Scanner;
+import java.io.IOException;
+
+/**
+ * Duke class provides the functionality for Duke chatbot.
+ *
+ * @author Rosa Kang
+ */
+public class Duke {
+    /**
+     * For the display of the text on the console.
+     */
+    protected Ui ui;
+    protected Storage s;
+    protected TaskList tasks;
+
+    /**
+     * Constructor for Duke.
+     */
```

Comment:
I feel that it would be better if there is a description of the parameter "path" here.